analyzers/src/SonarAnalyzer.CSharp/Rules/AvoidExcessiveInheritance.cs

```diff
@@ -36,89 +38,108 @@ namespace SonarAnalyzer.Rules.CSharp
     public class AvoidExcessiveInheritance : ParameterLoadingDiagnosticAnalyzer
     {
         private const string DiagnosticId = "S110";
-        private const string MessageFormat = "This class has {0} parents which is greater than {1} authorized.";
+        private const string MessageFormat = "This {0} has {1} parents which is greater than {2} authorized.";
```

Comment:
Not quite sure, but probably makes sense to state the change of the message here https://jira.sonarsource.com/browse/RSPEC-3914